ui/src/data/actions/index.ts

```diff
@@ -0,0 +1,18 @@
+export type Action =
+  | ReturnType<typeof resetCachedQueryResults>
+  | ReturnType<typeof setQueryResultsByQueryID>
+
+export const hashCode = s =>
```

Comment:
it's usually a good idea to paste the link you copy/pasted internet code form so people can get some context on it, and in the case of bugs, go right to the source.